app/javascript/layouts/Layout.tsx

```diff
@@ -157,7 +157,7 @@ const Layout = (props: LayoutProps) => {
           flattenSubMenus={true}
           imageOnly={true}
           mobileText={true}
-          logoWidth={"medium"}
+          logoWidth={"base"}
```

Comment:
the logo looks worse on this review app than on full. I don't think we need to change it?
This app:
![image](https://user-images.githubusercontent.com/11825994/141191641-b71c9319-868d-4aa6-add7-3ec6991b3b9e.png)
Full:
![Uploading image.png…]()